--- simmain.cc.orig	2022-02-09 21:46:02.992149485 +0100
+++ simmain.cc	2022-02-10 00:37:53.024194053 +0100
@@ -531,8 +531,11 @@
 		strcat( env_t::data_dir, PATH_SEPARATOR );
 	}
 	else {
-		strcpy( env_t::data_dir, argv[0] );
-		*(strrchr( env_t::data_dir, PATH_SEPARATOR[0] )+1) = 0;
+		static char buffer[1000];
+		sprintf(buffer, "%s/simutrans/", getenv("XDG_DATA_HOME"));
+		strcpy(env_t::data_dir, buffer);
+		// Make simutrans copy the installation when opened (workaound)
+		system("\cp -r /app/share/games/simutrans/ $XDG_DATA_HOME 2>/dev/null");
 
 #ifdef __APPLE__
 		// Detect if the binary is started inside an application bundle
